components/03-modules/minicart/Minicart.html

```diff
@@ -0,0 +1,140 @@
+<div class="minicart">
+  <alpaca-list class="minicart__list">
+    <alpaca-cart-list-item
+      v-for="{ key, item } in getImageWithKey"
+      :key="key"
+      item-tag="li"
+      :product-image="item.productImage"
+      :product-name="item.productName"
+      :product-price="item.price"
+      :product-id="item.productId"
+      :product-options="item.productOptions"
+      :quantity="item.quantity"
+      :name="`qty_${key}`"
+      href="#"
+    />
+  </alpaca-list>
+  <section class="minicart__summary">
+    <alpaca-heading
+      :level="3"
+      class="
+        heading
+        heading--first-level
+        minicart__summary-heading
+      "
+    >
+      {{ summaryTitle }}
+    </alpaca-heading>
+    <div
+      class="
+        row
+        row--no-margins
+        between-xs
+      "
+    >
+      <div class="col-xs-9">
+        Subtotal
+      </div>
+      <div
+        class="
+          col-xs-3
+          end-xs
+        "
+      >
+        $00.00
+      </div>
+    </div>
+    <div
+      class="
+        row
+        row--no-margins
+        between-xs
+      "
+    >
+      <div class="col-xs-9">
+        Tax
+      </div>
+      <div
+        class="
+          col-xs-3
+          end-xs
+        "
+      >
+        $00.00
+      </div>
+    </div>
+    <div
+      class="
+        row
+        row--no-margins
+        between-xs
+      "
+    >
+      <div class="col-xs-9">
+        Next long row here
+      </div>
+      <div
+        class="
+          col-xs-3
+          end-xs
+        "
+      >
+        $00.00
+      </div>
+    </div>
+    <div
+      class="
+        row
+        row--no-margins
+        between-xs
+        minicart__summary-row
+      "
+    >
+      <div class="col-xs-9">
+        <strong>Order Total</strong>
+      </div>
+      <div
+        class="
+          col-xs-3
+          end-xs
+        "
+      >
+        <strong>$00.00</strong>
+      </div>
+    </div>
+  </section>
```

Comment:
hmm the bast name is to mock these data in `totals` object

```
cart/totals:Array[2]
0:Object
code:"subtotalInclTax"
title:"Kwota częściowa brutto"
value:39.9
1:Object
code:"grand_total"
title:"Łączna suma"
value:39.9
```
sth like this

Reply:
Good idea, let me know if now it's better :slightly_smiling_face:

Reply:
much better ;)